File: jsonviewer/src/main/java/com/yuyh/jsonviewer/library/adapter/JsonItemViewHolder.java
package com.yuyh.jsonviewer.library.adapter;

import androidx.recyclerview.widget.RecyclerView;

import com.yuyh.jsonviewer.library.view.JsonItemView;

public class JsonItemViewHolder extends RecyclerView.ViewHolder {

    private JsonItemView itemView;

    JsonItemViewHolder(JsonItemView itemView) {
        super(itemView);
        setIsRecyclable(false);
        this.itemView = itemView;
    }

    public JsonItemView getItemView() {
        return itemView;
    }
}

File: jsonviewer/src/main/java/com/yuyh/jsonviewer/library/adapter/JsonItemViewHolder.kt
package com.yuyh.jsonviewer.library.adapter

import androidx.recyclerview.widget.RecyclerView
import com.yuyh.jsonviewer.library.view.JsonItemView

class JsonItemViewHolder internal constructor(val jsonItemView: JsonItemView)
    : RecyclerView.ViewHolder(jsonItemView) {
    init { setIsRecyclable(false) }
}